tests/test_hep_model.py

```diff
@@ -34,3 +37,33 @@ def test_special_collections_map_contains_all_valid_special_collections():
     result = SPECIAL_COLLECTIONS_MAP.keys()
 
     assert sorted(expected) == sorted(result)
+
+
+def test_curated():
+    schema = load_schema('hep')
+    subschema = schema['properties']['curated']
+
+    # Can't create a record from an empty snippet, so added this dummy content
```

Comment:
Not a completely empty snippet, because it has to be valid XML, and valid XML has at least a root tag, so you could use `<record></record>` instead (maybe `<record/> works?)